tests/integration/testdata/sync/create_new_service/kong.yaml

```diff
@@ -0,0 +1,18 @@
+_format_version: "1.1"
```

Comment:
I didn't catch this in the last review, it would be best to prefix these directories with a number: 001-create-new-service, that is much easier on the eyes and is easy to grep.